src/index.js

```diff
@@ -110,3 +110,5 @@ function addHandler(method, handlers, handler) {
 }
 
 module.exports = MockAdapter;
+// allow default import in typescript
+module.exports.default = MockAdapter;
```

Comment:
I think this can be just `module.exports = module.exports.default = MockAdapter`, unless there's a better way :)